Intelipro/reja
# Chore: Code Splitting

#### What does this PR do?
It implements code (bundle)splitting 

#### Description of Task to be completed?

- [x]  Implement lazy load in the views folder
- [x]  Implement a suspense callback with a loader for the lazy loaders

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ch-code-splitting`
- Install all dependencies `npm install`
- Start the server `npm run build`
-  You'll notice  multiple bundles have been created that can be dynamically loaded at runtime

#### Any background context you want to provide?
To avoid winding up with a large bundle, it’s good to get ahead of the problem and start “splitting” your bundle.
Code-Splitting is a feature which can create multiple bundles that can be dynamically loaded at runtime

#### Screenshots (if appropriate)

#### Questions:
